source/includes/administration/_usage.md

```diff
@@ -146,3 +145,133 @@ Query Parameters | &nbsp;
 `include_deleted`<br/>*boolean* | Will find usage of an organization that may have been deleted.
+
+## Retrieve service usage
+
+`GET /service/usage`
+
+```shell
+# Retrieve customer usage
+curl -X POST "https://cloudmc_endpoint/rest/service/usage?page=1&page_size=1" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+ "data": {
+  "records": [
+   {
+    "id": "a855e912f682b398379eefd5bbeffa2e:805",
+    "usageType": "memory_type",
+    "usage": 143.99833333333333,
+    "unit": "gibibyte-hour",
+    "connectionId": "081ffa1f-1531-4dd5-a7e2-070e560409ce",
+    "connectionName": "Edge Services",
+    "organizationId": "ca785487-5e22-4bac-ab01-187260b0d0cb",
+    "organizationName": "Cox",
+    "environmentId": "207ed06a-5b20-4d79-a7eb-d9ac4878519b",
+    "environmentName": "jdias-test-jason",
+    "startDate": "2022-01-01T00:00:00Z",
+    "endDate": "2022-01-02T00:00:00Z",
+    "fields": {
+     "PoP": "Phoenix",
+     "type": "Container",
+     "region": "NA"
+    },
+    "metricType": "GAUGE",
+    "metadata": {
+     "serviceCode": "edge-cloud-beta",
+     "account_id": "69b60c67-6175-4515-bb65-033108305d66",
+     "stackId": "cb9caf46-323f-402c-b173-742ba464e50e",
+     "service_account_id": "657c6671-872f-4590-8e31-a9bc6589918b",
+     "stackSlug": "jdias-test-jaso-7vhhr3lplg-env",
+     "client_id": "4e0c5f157c5d089671a57c4132205667",
+     "credential_id": "8b4e6d56-216b-4c70-8ce6-d283042aa3b9"
+    }
+   }
+  ],
+  "metadata": {
+   "totalCount": 13922,
+   "totalPageCount": 13923,
+   "count": 1,
+   "page": 1
+  }
+ }
+}
+```
+
+| Record Attributes                 | Description
+| ------------------                | ---------------
+| id <br> *String*                  | The id of the record.
+| usageType <br> *String*           | The type of the underlying usage.
+| usage <br> *Number*               | The amount of usage within this usage records period.
+| unit <br> *String*                | The unit attributed to the usage.
+| connectionId <br> *String*        | The id of the connection on which this usage was generated.
+| connectionName <br> *String*      | The name of the connection on which this usage was generated.
+| organizationId <br> *String*      | The id of the organization on which this usage was generated.
+| organizationName <br> *String*    | The name of the organization on which this usage was generated.
+| environmentId <br> *String*       | The id of the environment on which this usage was generated.
+| environmentName <br> *String*     | The name of the environment on which this usage was generated.
+| startDate <br> *ISO8601 Date Time*| The start date of the record (inclusive)
+| endDate <br> *ISO8601 Date Time*  | The end date of the record (exclusive)
+| fields <br> *Object*              | Attributes of the underlying resource that created this usage
+| metricType <br> *String*          | The type of usage specified. [COUNTER,GAUGE].
```

Comment:
This reads pretty similar to line 208.

Maybe we could re-write it to something like `The metric type of the usage` (I dont like this very much either)
but the `The type of usage specified` and `The type of the underlying usage` are pretty similar too

🤔

Reply:
Good point! I updated to add a bit more explanation: [`2b95766` (#507)](https://github.com/cloudops/cloudmc-api-docs/pull/507/commits/2b957661f68915db3aca88ad06e338db56eed4bf)